### PR TITLE
[projects] Fix rendering of startedAt attribute

### LINKED ISSUE #5518: Later prebuild timestamp is inaccurate on the project card
### Problem to solve

The latest prebuild timestamp information on the project cards is inaccurate. See screenshot below where a prebuild was triggerred _an hour ago_ while the latest prebuild timestamp reads _14 hours ago_.

| Example Card | Example Latest Peebuild |
|-|-|
| Screenshot 2021-09-03 at 1 46 21 PM | Screenshot 2021-09-03 at 1 46 23 PM (2) |

### PR DESCRIPTION
**Pull request by AlexTugarev**
## Description
Fixes rendering rendering of a timestamp

## Related Issue(s)

Fixes #5518

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **alextugarev** after the PR has been reviewed.

Associated issue: *#5518*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by jankeromnes — APPROVED** [review]
Code change looks good and works for me! (Layout issues for 'small 15" retina screens' aside 😋)

Screenshot 2021-10-01 at 09 30 41

/approve

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 82dc0734429384df02af5ceed3b1fb7863195bc8</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5961#pullrequestreview-768645276" title="Approved">jankeromnes</a>*

Associated issue: *#5518*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
**issue**: FYI, Prebuilds in this preview environment fail with the following error but maybe this is expected?

> Error: image build did not produce a workspace image

🍊 🍊 🍊 🍊 

Opened a follow up issue for https://github.com/gitpod-io/gitpod/issues/5978.

Cc @jankeromnes @AlexTugarev

**Comment by csweichel** [comment]
> **issue**: FYI, Prebuilds in this preview environment fail with the following error but maybe this is expected?
> 
> > Error: image build did not produce a workspace image
> 
> 🍊 🍊 🍊 🍊
> 
> Opened a follow up issue for #5978.
> 
> Cc @jankeromnes @AlexTugarev

No. This is an image-builder-mk3 issue. It turns out that it's really really hard to just copy an image from A to B :/ 
I've filed an issue: https://github.com/gitpod-io/gitpod/issues/5981

**Comment by gtsiolis** [comment]
Thanks for coming back to this and linking the new issue, @csweichel! 🏓

**Pull request merged**
